.github/workflows/CI.yml

```diff
@@ -9,6 +9,15 @@ on:
       - "README.md"
   pull_request:
 
+concurrency:
+  # Skip intermediate builds: all builds except for builds on the `master` branch
+  # Cancel intermediate builds: only pull request builds
+  group: ${{ github.workflow }}-${{ github.ref }}-${{ github.ref != 'refs/heads/main' || github.run_number }}
+  cancel-in-progress: ${{ startsWith(github.ref, 'refs/pull/') }}
```

Comment:
We don't need all of this in this repo given the low activity. Same with the timeouts. Could you either remove the changes or move them to a separate commit to keep the history cleaner, please?

Reply:
Moved to #5